### PR TITLE
Update ROS2 image and model

### PR DESCRIPTION
**Pull request by kan-bayashi**
## What?

- Update ROS2 image and model

## Why?

- Wanna to generate more metadata

```
(pydtk-py3.10) root@kan-bayashi-deploy-0:/opt/pydtk# pydtk model generate metadata --from-file sample/sample_autoware.db3
[WARN] [1674713541.242857171] [rosbag2_storage]: No storage plugin found with id 'sqlite3'.
[INFO] [1674713541.265295821] [rosbag2_storage]: Opened database 'sample/sample_autoware.db3' for READ_ONLY.
[WARN] [1674713541.437573104] [rosbag2_storage]: No storage plugin found with id 'sqlite3'.
[INFO] [1674713541.442175123] [rosbag2_storage]: Opened database 'sample/sample_autoware.db3' for READ_ONLY.
[WARN] [1674713541.613120382] [rosbag2_storage]: No storage plugin found with id 'sqlite3'.
[INFO] [1674713541.617680635] [rosbag2_storage]: Opened database 'sample/sample_autoware.db3' for READ_ONLY.
{
    "record_id": null,
    "path": "/opt/pydtk/sample/sample_autoware.db3",
    "contents": {
        "bag_size": 275415040,
        "compression_format": "",
        "compression_mode": "",
        "message_count": 8262,
        "duration": "0:00:29.874325",
        "starting_time": "2021-02-26 05:02:26.338218",
        "ending_time": "2021-02-26 05:02:56.212543",
        "storage_identifier": "sqlite3",
        "topics": {
            "/clock": {
                "message_count": 2941,
                "type": "rosgraph_msgs/msg/Clock",
                "serialization_format": "cdr",
                "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
            },
            "/sensing/gnss/ublox/fix_velocity": {
                "message_count": 30,
                "type": "geometry_msgs/msg/TwistWithCovarianceStamped",
                "serialization_format": "cdr",
                "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
            },
            "/sensing/gnss/ublox/nav_sat_fix": {
                "message_count": 30,
                "type": "sensor_msgs/msg/NavSatFix",
                "serialization_format": "cdr",
                "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
            },
            "/sensing/gnss/ublox/navpvt": {
                "message_count": 30,
                "type": "ublox_msgs/msg/NavPVT",
                "serialization_format": "cdr",
                "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
            },
            "/sensing/imu/tamagawa/imu_raw": {
                "message_count": 853,
                "type": "sensor_msgs/msg/Imu",
                "serialization_format": "cdr",
                "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
            },
            "/sensing/lidar/left/velodyne_packets": {
                "message_count": 299,
                "type": "velodyne_msgs/msg/VelodyneScan",
                "serialization_format": "cdr",
                "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
            },
            "/sensing/lidar/right/velodyne_packets": {
                "message_count": 299,
                "type": "velodyne_msgs/msg/VelodyneScan",
                "serialization_format": "cdr",
                "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
            },
            "/sensing/lidar/top/velodyne_packets": {
                "message_count": 288,
                "type": "velodyne_msgs/msg/VelodyneScan",
                "serialization_format": "cdr",
                "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
            },
            "/vehicle/status/control_mode": {
                "message_count": 873,
                "type": "autoware_auto_vehicle_msgs/msg/ControlModeReport",
                "serialization_format": "cdr",
                "offered_qos_profiles": ""
            },
            "/vehicle/status/gear_status": {
                "message_count": 873,
                "type": "autoware_auto_vehicle_msgs/msg/GearReport",
                "serialization_format": "cdr",
                "offered_qos_profiles": ""
            },
            "/vehicle/status/steering_status": {
                "message_count": 873,
                "type": "autoware_auto_vehicle_msgs/SteeringReport",
                "serialization_format": "cdr",
                "offered_qos_profiles": ""
            },
            "/vehicle/status/velocity_status": {
                "message_count": 873,
                "type": "autoware_auto_vehicle_msgs/msg/VelocityReport",
                "serialization_format": "cdr",
                "offered_qos_profiles": ""
            }
        }
    },
    "start_timestamp": 1614315746.338218,
    "end_timestamp": 1614315776.212543
}
```

### CODE REVIEW

File: pydtk/models/rosbag2.py [hunk 1]
```diff
@@ -329,20 +329,27 @@ def generate_contents_meta(cls, path, content_key="content"):
             dict: Contents metadata
 
         """
```
Comment (d-hayashi) [inline pydtk/models/rosbag2.py:331]: Please rollback this change to align the format of `contents` with other models.

**Comment by kan-bayashi** [comment]
Fix contents as follows:
```
root@kan-bayashi-deploy-0:/opt/pydtk# pydtk model generate metadata --from-file sample/sample_autoware.db3
[WARN] [1674734116.099633458] [rosbag2_storage]: No storage plugin found with id 'sqlite3'.
[INFO] [1674734116.105307128] [rosbag2_storage]: Opened database 'sample/sample_autoware.db3' for READ_ONLY.
[WARN] [1674734116.367407504] [rosbag2_storage]: No storage plugin found with id 'sqlite3'.
[INFO] [1674734116.372472992] [rosbag2_storage]: Opened database 'sample/sample_autoware.db3' for READ_ONLY.
[WARN] [1674734116.561068059] [rosbag2_storage]: No storage plugin found with id 'sqlite3'.
[INFO] [1674734116.566302425] [rosbag2_storage]: Opened database 'sample/sample_autoware.db3' for READ_ONLY.
{
    "record_id": null,
    "path": "/opt/pydtk/sample/sample_autoware.db3",
    "contents": {
        "/clock": {
            "message_count": 2941,
            "type": "rosgraph_msgs/msg/Clock",
            "serialization_format": "cdr",
            "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
        },
        "/sensing/gnss/ublox/fix_velocity": {
            "message_count": 30,
            "type": "geometry_msgs/msg/TwistWithCovarianceStamped",
            "serialization_format": "cdr",
            "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
        },
        "/sensing/gnss/ublox/nav_sat_fix": {
            "message_count": 30,
            "type": "sensor_msgs/msg/NavSatFix",
            "serialization_format": "cdr",
            "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
        },
        "/sensing/gnss/ublox/navpvt": {
            "message_count": 30,
            "type": "ublox_msgs/msg/NavPVT",
            "serialization_format": "cdr",
            "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
        },
        "/sensing/imu/tamagawa/imu_raw": {
            "message_count": 853,
            "type": "sensor_msgs/msg/Imu",
            "serialization_format": "cdr",
            "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
        },
        "/sensing/lidar/left/velodyne_packets": {
            "message_count": 299,
            "type": "velodyne_msgs/msg/VelodyneScan",
            "serialization_format": "cdr",
            "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
        },
        "/sensing/lidar/right/velodyne_packets": {
            "message_count": 299,
            "type": "velodyne_msgs/msg/VelodyneScan",
            "serialization_format": "cdr",
            "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
        },
        "/sensing/lidar/top/velodyne_packets": {
            "message_count": 288,
            "type": "velodyne_msgs/msg/VelodyneScan",
            "serialization_format": "cdr",
            "offered_qos_profiles": "- history: 3\n  depth: 0\n  reliability: 1\n  durability: 2\n  deadline:\n    sec: 2147483647\n    nsec: 4294967295\n  lifespan:\n    sec: 2147483647\n    nsec: 4294967295\n  liveliness: 1\n  liveliness_lease_duration:\n    sec: 2147483647\n    nsec: 4294967295\n  avoid_ros_namespace_conventions: false"
        },
        "/vehicle/status/control_mode": {
            "message_count": 873,
            "type": "autoware_auto_vehicle_msgs/msg/ControlModeReport",
            "serialization_format": "cdr",
            "offered_qos_profiles": ""
        },
        "/vehicle/status/gear_status": {
            "message_count": 873,
            "type": "autoware_auto_vehicle_msgs/msg/GearReport",
            "serialization_format": "cdr",
            "offered_qos_profiles": ""
        },
        "/vehicle/status/steering_status": {
            "message_count": 873,
            "type": "autoware_auto_vehicle_msgs/SteeringReport",
            "serialization_format": "cdr",
            "offered_qos_profiles": ""
        },
        "/vehicle/status/velocity_status": {
            "message_count": 873,
            "type": "autoware_auto_vehicle_msgs/msg/VelocityReport",
            "serialization_format": "cdr",
            "offered_qos_profiles": ""
        }
    },
    "start_timestamp": 1614315746.338218,
    "end_timestamp": 1614315776.212543
}
```

File: pydtk/models/rosbag2.py (hunk 1 above)
Comment (kan-bayashi) [reply pydtk/models/rosbag2.py:331]: fixed

**Review by d-hayashi — APPROVED** [review]
LGTM

**Pull request merged**